ee/tabby-webserver/src/service/repository/mod.rs

```diff
@@ -105,15 +105,15 @@ impl RepositoryService for RepositoryServiceImpl {
     }
 
     async fn resolve_repository(&self, kind: &RepositoryKind, id: &ID) -> Result<Repository> {
-        if let RepositoryKind::Git = kind {
+        if let RepositoryKind::GitConfig = kind {
             if let Ok(index) = config_id_to_index(id) {
                 let config = &self.config[index];
                 return repository_config_to_repository(index, config);
             }
         }
 
         match kind {
-            RepositoryKind::Git => self.git().get_repository(id).await,
+            RepositoryKind::Git | RepositoryKind::GitConfig => self.git().get_repository(id).await,
```

Comment:
will this be working? can you test this behavior locally?

see https://tabby.tabbyml.com/docs/administration/context/#adding-a-repository-through-configuration-file for a `config.toml` based configuration, and verify it in `Code Browser`